### PR TITLE
:sparkles: ✨ feature Issues 559 and 560 - Stop using KCP's Location and SyncTarget API resources

### PR DESCRIPTION
**Pull request by fileppb**
## Summary
Stop using KCP's Location and SyncTarget API resources

## Related issue(s)
Issue 559
Issue 560 

Fixes #

### CODE REVIEW

**Comment by fileppb** [comment]
Weirdly this is showing merge conflicts in the where-resolver which isn't even in my source files.  see: 
 CONFLICT (content): Merge conflict in pkg/where-resolver/controller.go
Auto-merging cmd/kubestellar-where-resolver/cmd/where-resolver-cmd.go
CONFLICT (content): Merge conflict in cmd/kubestellar-where-resolver/cmd/where-resolver-cmd.go
Automatic merge failed; fix conflicts and then commit the result. 

And I don't see the cmd/kubestellar-schedule/cmd/scheduler.go, pkg/scheduler/controller.go etc conflicts in the prow reports or when I try to rebase

File: cmd/kubestellar-scheduler/cmd/scheduler.go [hunk 1]
```diff
@@ -159,8 +163,11 @@ func Run(ctx context.Context, options *scheduleroptions.Options) error {
 		edgeClusterClientset,
 		edgeSharedInformerFactory.Edge().V1alpha1().EdgePlacements(),
 		edgeSharedInformerFactory.Edge().V1alpha1().SinglePlacementSlices(),
-		schedulingSharedInformerFactory.Scheduling().V1alpha1().Locations(),
-		workloadSharedInformerFactory.Workload().V1alpha1().SyncTargets(),
+		// should these next two lines be commented out? Or should we still allow kcp objects?
```
Comment (MikeSpreitzer) [inline cmd/kubestellar-scheduler/cmd/scheduler.go:166]: Yes, commented out.  We want to stop using the object kind definitions found in the kcp repo.
You should be getting warnings that schedulingSharedInformerFactory and workloadSharedInformerFactory are not used; you can entirely remove them.

**Comment by waltforme** [comment]
@clubanderson Sure, looks like the conflicts are because of #719 was merged after the commit at which this PR branches apart from main. So during the rebasing, we should accept those changes introduced by #719 (which includes renaming of several files).

**Comment by fileppb** [comment]
@waltforme @clubanderson I've been spending the past 2 hours re-basing from upstream/main. I think I've resolved the conflicts and completed the rebase. But I'd like to run a test of the Quickstart steps again using the rebased code. So when I get back from walking the dogs I'll do that. Then I guess I'll commit to my git origin and see what happens with the PR. Is there something I should be doing differently???

**Comment by clubanderson** [comment]
a commit to your origin should update this PR and clear up the rebase.  lets see what happens

**Comment by clubanderson** [comment]
@fileppb I do not see a new commit from you on this PR so a retest won't help here.

**Comment by fileppb** [comment]
well one issue is that I'm trying to execute the PR from my main, instead of the branch i've been working in. So i think I may close this PR and open a new one from the correct branch.

**Comment by clubanderson** [comment]
yeah - that won't work - start over.

**Comment by fileppb** [comment]
PR was based on wrong branch, will open new PR

**Pull request closed without merging**